src/App.js

```diff
@@ -3,17 +3,51 @@ import Header from './components/Header.js';
 import Gallery from './components/Gallery.js';
 import React from 'react';
 import pokeArray from './data/poke-data.js';
+import SearchDashboard from './components/SearchDashboard';
 
 export default class App extends React.Component {
 
   state = {
     displayItems: pokeArray,
+    searchBarState: '',
+    sortDirectionState: '',
   }
+
+  searchBarChange = (e) => {
+    this.setState({
+      searchBarState: e.target.value,
+    })
+  }
+
```

Comment:
make a submit submitHandle and set it to the state that controls your onchange for the input. 
it'll be like this.setState({
      submitState: this.state.searchBarState (or whatever is the input onChange state)
})